### PR TITLE
Add button to request password reset

### PR DESCRIPTION
**Pull request by Devessier**

### CODE REVIEW

File: packages/client/services/AuthenticationService.ts [hunk 1]
```diff
@@ -214,3 +215,28 @@ export async function sendResendingConfirmationEmail(): Promise<ResendConfirmati
...
+}: SendRequestingPasswordResetArgs): Promise<RequestPasswordResetResponseBody> {
+    const requestBody: RequestPasswordResetRequestBody = {
+        email,
+    };
+    const rawResponse = await request.post(
+        '/authentication/request-password-reset',
+        requestBody,
+        {
+            validateStatus: (status) =>
+                status === 200 || status === 404 || status === 429,
```
Comment (prastoin) [inline packages/client/services/AuthenticationService.ts:234]: praise: I think we should always be doing something like that !

Comment (prastoin) [reply packages/client/services/AuthenticationService.ts:234]: We never knows what exception adonis could throw between 200 and 429

File: packages/client/screens/SigningInScreen.tsx [hunk 2]
```diff
@@ -223,6 +240,17 @@ const SigningInScreen: React.FC<SigningInScreenProps> = ({ navigation }) => {
...
                                 />
 
+                                <TouchableOpacity
+                                    style={sx({
+                                        marginTop: 'm',
+                                    })}
+                                    onPress={handleRequestPasswordReset}
+                                >
+                                    <Text sx={{ color: 'white' }}>
+                                        Do you lost your password?
```
Comment (prastoin) [inline packages/client/screens/SigningInScreen.tsx:250]: nitpick: Unless I'm mistaken should this be `Did you lost your password ?` or maybe we could be saying something like `Reset password`

Comment (Devessier) [reply packages/client/screens/SigningInScreen.tsx:250]: I made a typo you're right, but I think it should be _Did you lose your password_? Or _Have you lost your password_?

Comment (prastoin) [reply packages/client/screens/SigningInScreen.tsx:250]: Indeed :laughing:

**Pull request merged**